### ISSUE TITLE
Integrate help commands into RPC

### ISSUE THREAD

**Issue by Varunram** [mit-dci/lit#246]
Right now, the help commands are all over the place. It brings value to consolidate all of them in one place and allow them to be called from there. This is especially true for lit-af, where most commands need interaction from inside the interface. This could also be done via a separate CLI for `lit` where the CLI thing does e everything lit-af does, but via the command line directly without requiring direct interaction with `lit-af`. Also could help the doc gen stuff since that's hardcoded now.

**Comment by delbonis** [comment]
Could we have lit-af just directly run the command if it's given something like a `-c` option and then the command (a la Bash)?  It seems a bit redundant to make another program that does one-offs.

**Comment by Varunram** [comment]
Yeah, I guess calling bash and executing lit-af from there should work. But would that be a separate arg to lit that when passed executes the shell script?

**Comment by delbonis** [comment]
I meant how if you say `bash -c cat /etc/passwd`, it'll tell Bash to run that command.  I'm suggesting that if you run `lit-af -c lis 1337` it'll invoke *that* command instead of starting the normal lit-af shell.

**Comment by Varunram** [comment]
Yes, I got that but it falls back to another method for hardcoding the args since I'd have to call all the lit-af commands individually :(

**Comment by delbonis** [comment]
Wait so do you want to fork Bash?
